daffodil-tdml/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -323,20 +342,20 @@ class DFDLTestSuite private[tdml] (
     }
   }
 
-  def runPerfTest(testName: String, schema: Option[Node] = None) {
-    var bytesProcessed: Long = 0
-    Tak.calibrate
-    val ns = Timer.getTimeNS(testName, {
-      val nBytes = runOneTestWithDataVolumes(testName, schema)
-      bytesProcessed = nBytes
-    })
-    // val takeonsThisRun = ns / Tak.takeons
-    val bpns = ((bytesProcessed * 1.0) / ns)
-    val kbps = bpns * 1000000
-    val callsPerByte = 1 / (Tak.takeons * bpns)
-    println("\nKB/sec = " + kbps)
-    println("tak call equivalents per byte (takeons/byte) =  " + callsPerByte)
-  }
+  //  def runPerfTest(testName: String, schema: Option[Node] = None) {
```

Comment:
Should this function just be removed, or was this the function that was being used for the overnight performance testing?